packages/preview-middleware-client/src/adp/quick-actions/fe-v2/create-table-custom-column.ts

```diff
@@ -61,12 +61,19 @@ export function preprocessActionExecution(
 
 export class AddTableCustomColumnQuickAction
     extends TableQuickActionDefinitionBase
-    implements NestedQuickActionDefinition
-{
+    implements NestedQuickActionDefinition {
     constructor(context: QuickActionContext) {
         super(CREATE_TABLE_CUSTOM_COLUMN, CONTROL_TYPES, 'QUICK_ACTION_ADD_CUSTOM_TABLE_COLUMN', context);
     }
 
+    async initialize(): Promise<void> {
+        await super.initialize((table, child) => {
+            this.initializeCustomColumnTable(table, child);
+        });
+    }
+
```

Comment:
Why linter allows to have many empty lines? Did you run pnpm lint?

Reply:
yes, I did , and didn't get any issue.

Reply:
which spaces you mean ? I can manually remove

Reply:
manually removed.

Reply:
there was some incorrect formatting setting. I fixed and now it works..